tests/examples/migrations/preg_with_matches.hack.in

```diff
@@ -0,0 +1,44 @@
+<?hh // strict
+
+/**
+ * Copyright (c) 2016, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional
+ * grant of patent rights can be found in the PATENTS file in the same
+ * directory.
+ *
+ */
+
+function foo(): void {
+  $matches = null;
+
+  \preg_match('/foo/', 'bar');
+  \preg_match_all('/foo/', 'bar');
+
+  \preg_match('/foo/', 'bar', &$matches);
+  \preg_match_all('/foo/', 'bar', &$matches);
+
+  \preg_match('/foo/', 'bar', inout $matches);
+  \preg_match_all('/foo/', 'bar', inout $matches);
+
+  \preg_match('/foo/', 'bar', inout $matches, \PREG_UNMATCHED_AS_NULL, 42);
+  \preg_match_all(
+    '/foo/',
+    'bar',
+    &$matches,
+    \PREG_SET_ORDER | \PREG_OFFSET_CAPTURE,
+  );
+
+  \ // Weird formatting is preserved.
+  preg_match /* So long and thanks for all the fish. */ (
+    '/foo/',
+    'bar',
+    &/* no space here */$matches,
+  );
+
+  // Not in the root namespace.
+  preg_match('/foo/', 'bar', &$matches);
```

Comment:
This looks like it is in the root namespace to me - what's the fullly qualified name of the function it's meant to be calling?

Reply:
This was meant to be a "function in the current namespace" but I forgot to declare the current namespace in the file.